h/cli/__init__.py

```diff
@@ -37,13 +37,6 @@ def bootstrap(app_url, dev=False, create_db=False):
     # up an entire application to build the extensions.
     os.environ['H_SCRIPT'] = 'true'
 
-    # Override other important environment variables
-    os.environ['MODEL_CREATE_ALL'] = 'True' if create_db else 'False'
-    os.environ['MODEL_DROP_ALL'] = 'False'
-
-    if dev:
-        os.environ['SECRET_KEY'] = 'notsecret'
```

Comment:
This is no longer needed as all these commands now bootstrap against `development-app.ini` when appropriate, which already defines a dummy secret key for development.